modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1158,6 +1158,68 @@ definition {
 		}
 	}
 
+	@description = "LPS-165994 - Verify that the user cannot insert an invalid date on Start / End fields when creating a filter"
+	@priority = "5"
+	test CannotInsertInvalidDate {
+		property portal.acceptance = "true";
+
+		task ("Given an object is created with a date field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 165994",
+				objectName = "CustomObject165994",
+				pluralLabelName = "Custom Objects 165994");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 165994");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Field",
+				fieldType = "Date");
+
+			Refresh();
+
+			ObjectAdmin.assertObjectField(
+				fieldLabel = "Custom Field",
+				fieldType = "Date");
+
+			Refresh();
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 165994");
```

Comment:
If you are publishing the object via API, you can remove these steps that navigate to the objects portlet.